src/main/java/com/hazelcast/remotecontroller/RemoteControllerHandler.java

```diff
@@ -148,7 +148,7 @@ public Response executeOnController(String clusterId, String script, Lang lang)
         if (engine == null) {
             throw new IllegalArgumentException("Could not find ScriptEngine named:" + engineName);
         }
-        if (clusterId != null) {
+        if (clusterId != null && !clusterId.isEmpty()) {
```

Comment:
Can you add an explanation as a comment here?

Reply:
Fixed.